### PR TITLE
Add talk Managing multiple Kubernetes clusters by Jacob Larfors

### PR DESCRIPTION
**Pull request by jlarfors**

### CODE REVIEW

File: talks/jacob-larfors.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+**Linkedin (optional):** @jlarfors
+
+**Github (optional):** @jlarfors
+
+## About Talk
+
+**Meetup Group**: DevOps Malmö
+
+**Date**: May 2022 (to be confirmed)
```
Comment (gangsta) [inline talks/jacob-larfors.md]: Hi @jlarfors ,

do you have an exact date in your mind ? or waiting for confirmation from other parties?

Comment (jlarfors) [reply talks/jacob-larfors.md]: Yo @gangsta ! I think @kvarak had mentioned either Monday 2nd or Wednesday 4th May as available dates, based on venue availability. I can make either of those... so I'd say whichever day is "better" for the event, let's pick that day.

Comment (gangsta) [reply talks/jacob-larfors.md]: Yo @jlarfors let's do Wednesday the 4th :)

I would suggest adding this into `talks/jacob-larfors.md`  and in Readme file

as soon you have it ready will start organizing it :pray:

Comment (jlarfors) [reply talks/jacob-larfors.md]: @gangsta It is done. Once merged I'll book my travel 🚀

**Review by gangsta — APPROVED** [review]

File: talks/jacob-larfors.md (hunk 1 above)
Comment (gangsta) [reply talks/jacob-larfors.md]: Awesome :tada: looking forward to it. :handshake:

**Pull request merged**